android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/EditTextFactory.java

```diff
@@ -274,4 +279,24 @@ private void addNumericIntegerValidator(JSONObject jsonObject, MaterialEditText
             }
         }
     }
+
+    private void addRelativeNumericIntegerValidator(JSONObject editTextJSONObject, JsonFormFragment formFragment, MaterialEditText editText) throws JSONException {
+        JSONObject numericIntegerObject = editTextJSONObject.optJSONObject(JsonFormConstants.V_NUMERIC_INTEGER);
+        if (numericIntegerObject != null) {
+            String numericValue = numericIntegerObject.optString(JsonFormConstants.VALUE);
+            if (!TextUtils.isEmpty(numericValue) && Boolean.TRUE.toString().equalsIgnoreCase(numericValue)
```

Comment:
Refactor this to check that the field has defined the dependent field that has the maximum.

Also check that the field is actually defined in the form

Reply:
Done.